src/stack/Stack.java

```diff
@@ -19,16 +19,21 @@ public Stack(int size) {
 	 * Add element into stack
 	 */
 	public void push(int value) {
-
+		tos = tos + 1;
+		if (tos < data.length) {
+		data[tos] = value;
+		}
 	}
 
 	/**
 	 * Get element from the stack
 	 * @return
 	 */
-	public int pop() {
+	public int pop(){
+		int element = data[tos];
```

Comment:
You have to return the last element of the array and remove it from there. If the array is already empty show some message.